public/pages/CreateDetector/components/ConfigureAlerts/components/AlertCondition/AlertConditionPanel.tsx

```diff
@@ -56,7 +56,6 @@ interface AlertConditionPanelState {
   showNotificationDetails: boolean;
   detectionRulesTriggerEnabled: boolean;
   threatIntelTriggerEnabled: boolean;
-  notificationError: string;
```

Comment:
Would it still be useful to have the `notificationError` in some cases? Or was the only error message something like "Notification required"?

Reply:
That was the only error we had, so better to revert this change and add it later when we know which errors we want to cover

Reply:
Agreed. Thanks for clarifying.